src/pages/iou/MoneyRequestDescriptionPage.js

```diff
@@ -40,18 +40,22 @@ class MoneyRequestDescriptionPage extends Component {
         super(props);
 
         this.updateComment = this.updateComment.bind(this);
+        this.navigateBack = this.navigateBack.bind(this);
         this.iouType = lodashGet(props.route, 'params.iouType', '');
         this.reportID = lodashGet(props.route, 'params.reportID', '');
     }
 
     // eslint-disable-next-line rulesdir/prefer-early-return
     componentDidMount() {
         if (_.isEmpty(this.props.iou.participants) || this.props.iou.amount === 0) {
-            Navigation.goBack();
-            Navigation.navigate(ROUTES.getMoneyRequestRoute(this.iouType, this.reportID));
+            Navigation.goBack(ROUTES.getMoneyRequestRoute(this.iouType, this.reportID));
```

Comment:
I intentionally let it to be false. Imagine this case (in native):
1. User open money request
2. User fill amount and arrive in participant selector
3. User open deep link to confirmation (/request/new/confirmation)
4. Because participant is empty, it will go back to participant selector page

If we force the fallback, it will go to amount page instead. I think the experience is better. What do you think?

Reply:
I was thinking the other way around where navigating back is unwanted. I couldn't come up with a valid case yet, and it seems to work well as is, so :+1: let's mark this as resolved.